package.json

```diff
@@ -3,6 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.11.1",
+    "@emotion/styled": "^11.11.0",
+    "@mui/joy": "^5.0.0-beta.0",
```

Comment:
So far, the other front end members are using just MaterialUI instead of the sister library JoyUI. We should stick to that for now so that all of the UI looks consistent unless we find a specific component or something thats only available in JoyUI